test/test_generated_mypy.py

```diff
@@ -351,18 +351,16 @@ def test_extensions_proto2():
     e3.flag = True
     assert isinstance(e3, SeparateFileExtension)
 
-    # Currently any extension is typed as an Any.
-    e3 = s1.Extensions[Extensions1.ext]
-    e3 = 3.1415
+    del s1.Extensions[Extensions2.foo]
 
-    # Currently we don't check that the extension is being applied to the
-    # correct base model. This fails at runtime but passes mypy.
+    for x in s1.Extensions:
+        assert isinstance(x, FieldDescriptor)
+        y = s1.Extensions[x]  # y should be typed as an AnyMessage
```

Comment:
This comment might leave some confusion as to the why, I think more specifically, explaining what "x" should be typed as will help understand why "y" is an `Any` here.

Specifically, to add clarity that `y` is `Any` because `x` came out of an iterator, which can't know the value type.